lib/Controller/MentionController.php

```diff
@@ -39,14 +41,20 @@ public function mention(int $fileId, string $mention): DataResponse {
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
-		$userFolder = $this->rootFolder->getUserFolder($mention);
+		$userResults = $this->userManager->searchDisplayName($mention, 1);
+		if (count($userResults) < 1) {
+			return new DataResponse([], Http::STATUS_NOT_FOUND);
+		}
+
+		$user = $userResults[0];
```

Comment:
if you want to go safe, use `array_key_first()` or `array_shift` instead of the hard `0` index.